### PR TITLE
Redesign branch

### LINKED ISSUE #119: SettingsPane throws error if a project without a saved rate limiter configuration is selected.
Opening the settings pane for a proejct without a saved rate limiter configuration throws an undefine error when trying to access the `options` property and then displays a blank page to the user.

### PR DESCRIPTION
**Pull request by milos381**
### Summary

Final design suggestions\

### Type of Change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)

### Issues
- closes #119 

### Evidence
- UI was tested for responsiveness and functionality

### CODE REVIEW

File: client/App.tsx [hunk 1]
```diff
@@ -1,65 +1,163 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import settings from '../public/settings.png';
 import intime from '../public/intime.png';
 import data from '../public/data.png';
 import barchart from '../public/barchart.png';
 import snippet from '../public/code-snippet.png';
+import logo from '../public/logo.png';
```
Comment (shalarewicz) [inline client/App.tsx:7]: Rename the file in the public folder from `Logo.PNG` to `logo.png`. Can you also remove any images that aren't being used?

File: client/App.tsx [hunk 2]
```diff
@@ -1,65 +1,163 @@
...
-                            className="modest-vid-embed__item"
-                            style={{ width: '100%', height: '130%' }}
-                            src="https://www.youtube.com/embed/BQ0mxQXmLsk?mute=1&autoplay=1&modestbranding=1&loop=1&rel=0&amp;controls=0&amp;showinfo=0&playlist=rwlzvGfGXn4"
-                            frameBorder="0"
-                            allowFullScreen
-                        />
-                    </div>
                     <div id="pageOneTextBox">
-                        <h1>GraphQLGate</h1>
+                        <h1>GraphQL Gate</h1>
```
Comment (evanmcneely) [inline client/App.tsx:97]: I'm thinking one word, `GraphQLGate`, is more consistent with the writing everywhere else. Thoughts on this?

**Comment by evanmcneely** [comment]
The line doesn't extend all the way when dashboard/login is up.

Screen Shot 2022-08-06 at 6 19 28 PM

**Comment by evanmcneely** [comment]
I finish review later. Great work though.

File: client/components/ToolBar.tsx [hunk 3]
```diff
@@ -91,6 +90,7 @@ export default function ToolBar({
                             }
                         }}
                     />
+                    <h3 className="settingsTab">Settings:</h3>
```
Comment (shalarewicz) [inline client/components/ToolBar.tsx:93]: I think it looks better without this in here. The individual pane's have headings in them.

File: client/components/Team.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default function Team() {
+    return (
+        <div className="container">
+            <section id="aboutSection">
+                <div id="aboutDiv">
+                    <h1>OUR STORY</h1>
+                    <p>
+                        <strong style={{ color: '#2785d7ca' }}>GraphQL GATEWAY</strong> tool seriers
```
Comment (shalarewicz) [inline client/components/Team.tsx:10]: See drive for suggestions on updated wording.

File: client/components/Footer.tsx [hunk 5]
```diff
@@ -12,20 +12,20 @@ export default function Footer() {
...
                             : pathname === '/dashboard'
-                                ? '#fff'
-                                : '#092173',
+                                ? '#2785d7ca'
+                                : '#2785d7ca',
                 color:
                     // eslint-disable-next-line no-nested-ternary
                     pathname === '/signup'
-                        ? '#092173'
+                        ? '#fff'
```
Comment (shalarewicz) [inline client/components/Footer.tsx:24]: This always evaluates to `#fff`. Can it be simplified?

Comment (shalarewicz) [reply client/components/Footer.tsx:24]: Similar comment for `backgroundColor`

File: client/App.tsx (hunk 2 above)
Comment (shalarewicz) [inline client/App.tsx:97]: Revert to GraphQLGate

File: client/App.tsx [hunk 6]
```diff
@@ -1,65 +1,163 @@
...
+import logo from '../public/logo.png';
 import '../public/styles.css';
 
 export default function HomePage() {
+    const [loopNum, setLoopNum] = useState(0);
+    const [isDeleting, setIsDeleting] = useState(false);
+    const [text, setText] = useState('');
+    const [delta, setDelta] = useState(300 - Math.random() * 100);
+    const [index, setIndex] = useState(1);
+    const toRotate = ['GraphQL Gateway developer portal'];
```
Comment (shalarewicz) [inline client/App.tsx:16]: I think this can just save "Developer Portal"

**Review by shalarewicz — CHANGES_REQUESTED** [review]
The redesign looks super clean. Awesome work!

File: client/components/Team.tsx (hunk 4 above)
Comment (evanmcneely) [reply client/components/Team.tsx:10]: I agree, the wording at the bottom of the google doc is crisp.

**Review by evanmcneely — APPROVED** [review]
Address Stephan's comments and I'm happy. Great work to both of for coming together and finishing this off. I know it's taken extra effort at the end of a huge push but I think it paid off.

**Comment by evanmcneely** [comment]
for reference, this is the error causing the build to fail

```
ERROR in ./client/App.tsx 19:0-38
[1] Module not found: Error: Can't resolve '../public/logo.png' in '/home/travis/build/oslabs-beta/GraphQL-Gateway/client'
[1] resolve '../public/logo.png' in '/home/travis/build/oslabs-beta/GraphQL-Gateway/client'
[1]   using description file: /home/travis/build/oslabs-beta/GraphQL-Gateway/package.json (relative path: ./client)
[1]     Field 'browser' doesn't contain a valid alias configuration
[1]     using description file: /home/travis/build/oslabs-beta/GraphQL-Gateway/package.json (relative path: ./public/logo.png
[1]       no extension
[1]         Field 'browser' doesn't contain a valid alias configuration
[1]         /home/travis/build/oslabs-beta/GraphQL-Gateway/public/logo.png doesn't exist
[1]       .tsx
[1]         Field 'browser' doesn't contain a valid alias configuration
[1]         /home/travis/build/oslabs-beta/GraphQL-Gateway/public/logo.png.tsx doesn't exist
[1]       .ts
[1]         Field 'browser' doesn't contain a valid alias configuration
[1]         /home/travis/build/oslabs-beta/GraphQL-Gateway/public/logo.png.ts doesn't exist
[1]       .js
[1]         Field 'browser' doesn't contain a valid alias configuration
[1]         /home/travis/build/oslabs-beta/GraphQL-Gateway/public/logo.png.js doesn't exist
[1]       as directory
[1]         /home/travis/build/oslabs-beta/GraphQL-Gateway/public/logo.png doesn't exist
```

File: client/App.tsx (hunk 1 above)
Comment (evanmcneely) [reply client/App.tsx:7]: This is probably the reason the build is failing.

File: client/App.tsx (hunk 2 above)
Comment (shalarewicz) [reply client/App.tsx:97]: Agreed

File: client/App.tsx (hunk 1 above)
Comment (milos381) [reply client/App.tsx:7]: Done!

File: client/components/ToolBar.tsx (hunk 3 above)
Comment (milos381) [reply client/components/ToolBar.tsx:93]: Deleted.

File: client/components/Team.tsx (hunk 4 above)
Comment (milos381) [reply client/components/Team.tsx:10]: Done.

File: client/components/Footer.tsx (hunk 5 above)
Comment (milos381) [reply client/components/Footer.tsx:24]: Yes. Before the redesign, it made sense, due to the constant color changes. I will proceed and delete unnecessary  code.

File: client/App.tsx (hunk 2 above)
Comment (milos381) [reply client/App.tsx:97]: Done

File: client/App.tsx (hunk 6 above)
Comment (milos381) [reply client/App.tsx:16]: Ok

File: client/App.tsx [hunk 7]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (shalarewicz) [inline client/App.tsx:1]: Remove this comment. If there is an unused variable, evaluate whether or not we need it and then update the code accordingly.

**Review by shalarewicz — CHANGES_REQUESTED** [review]
The build was still failing due to the `Logo.PNG` filename. I've updated this filename and also removed comments disabling typescript / eslint rules. In general, we should avoid silencing these warnings for entire files. Warning will not cause a build failure. 

There is one outstanding task in `App.tsx` to determine if an unused variable is needed.

File: client/App.tsx (hunk 7 above)
Comment (milos381) [reply client/App.tsx]: I understand, it was an oversight.

**Review by donjewey — APPROVED** [review]

**Review by shalarewicz — APPROVED** [review]

**Pull request merged**